### PR TITLE
fix: user guide for cardinalities (DEV-1568)

### PR DESCRIPTION
**Pull request by danielasubotic**

### CODE REVIEW

File: docs/user-guide/project.md [hunk 1]
```diff
@@ -156,6 +156,36 @@ If you want to define a property which can have more than one value, you should
...
+
+### Widening or restricting a cardinality of a property if a resource does not contain all the values
+If values are missing in the resources and you want to adjust the cardinality so that the resources do not match the selected cardinality (e.g. set a property to required, but the field is empty), an error message is displayed. In this case, all resources must be corrected first, then the desired cardinality can be set.
+
+### Widening or restricting a cardinality of a property: Special cases
+#### Superclasses and subclasses
+If the cardinality of a superclass is changed, then the subclass is not affected and vice versa the same applies. In general, the cardinality of a subclass can be more or equally restrictive than the cardinality of a superclass.
+
+#### Inherited properties with different naming
+Changing the cardinalities of a superclass and subclass and reversing them does not work if the inherited properties are named differently.
```
Comment (Rebeccasigloch) [inline docs/user-guide/project.md:188]: Result of the discussion with Christian: Changing the cardinality of a superclass and a subclass and reversing it does not work if the properties are in a subproperty relationship or the changed property in the superclass does not exist in the subclass.

**Review by Rebeccasigloch — CHANGES_REQUESTED** [review]
Thanks for updating the sentence

File: docs/user-guide/project.md (hunk 1 above)
Comment (danielasubotic) [reply docs/user-guide/project.md]: thx! Done.

**Review by Rebeccasigloch — APPROVED** [review]
Looks good to me

**Pull request merged**